config/locales/en.yml

```diff
@@ -47,13 +47,15 @@ en:
       trips: "Trips"
   families:
     index:
-      title: "Families"
       add_family: "Add a Family"
-      destroy: "Destroy"
-      edit: "Update"
-      submit: "Add"
       destroy: "Delete"
       edit: "Edit"
+      active_families: "Families Active"
+      not_active_families: "Families Not Active"
+      all_families: "All families"
+      button_active_families: "Active"
+      button_not_active_families: "Not Active"
+      button_all_families: "All"
```

Comment:
What do you think about nesting the buttons under a `buttons` key? i.e.
```
families:
  index:
    buttons:
      active:
      inactive:
      all:
```

Reply:
Done in sha `3e97938`